hue/hue/base/hue-mysql-dc.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: hue-mysql
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
+spec:
+  strategy:
+    type: Recreate
+  triggers:
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+          - mysql
+        from:
+          kind: ImageStreamTag
+    - type: ConfigChange
+  replicas: 1
+  selector:
+    deploymentconfig: hue-mysql
+  template:
+    metadata:
+      labels:
+        deploymentconfig: hue-mysql
+    spec:
+      containers:
+        - name: mysqld-exporter
+          image: mysqld-exporter
+          ports:
+            - containerPort: 9104
+              protocol: TCP
+          env:
+            - name: DATA_SOURCE_NAME
+              valueFrom:
+                secretKeyRef:
+                  name: "hue-mysqld-exporter-secret"
+                  key: DATA_SOURCE_NAME
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 9104
+          readinessProbe:
+            httpGet:
+              path: /metrics
+              port: 9104
+        - name: mysql
+          image: mysql
+          ports:
+            - containerPort: 3306
+              protocol: TCP
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - "-i"
+                - "-c"
+                - >-
+                  MYSQL_PWD=$MYSQL_PASSWORD mysql -h 127.0.0.1 -u $MYSQL_USER -D
+                  $MYSQL_DATABASE -e 'SELECT 1'
+            failureThreshold: 3
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 1
+          livenessProbe:
+            failureThreshold: 3
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            successThreshold: 1
+            tcpSocket:
+              port: 3306
+            timeoutSeconds: 1
+          env:
+            - name: MYSQL_USER
+              valueFrom:
+                secretKeyRef:
+                  name: "hue-mysql-secret"
+                  key: database-user
+            - name: MYSQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "hue-mysql-secret"
+                  key: database-password
+            - name: MYSQL_DATABASE
+              valueFrom:
+                secretKeyRef:
+                  name: "hue-mysql-secret"
+                  key: database-name
+            - name: MYSQL_ROOT_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: "hue-mysql-secret"
+                  key: database-root-password
+          resources:
+            limits:
```

Comment:
I think we should set both limits and requests, otherwise (IIRC) if only limits is set, requests==limits

Reply:
@rimolive do you have any suggestion on what would be reasonable values here?

Reply:
I've just put some random values in there that seemed believable and sensible to me.